stdlib/Compflags

```diff
@@ -31,10 +31,11 @@ case $1 in
   stdlib__Printf.cm[io]|stdlib__Format.cm[io]|stdlib__Scanf.cm[io])
       echo ' -w +A -w -fragile-match';;
   stdlib__Scanf.cmx) echo ' -inline 9';;
-  *Labels.cmi) echo ' -pp "$AWK -f ./expand_module_aliases.awk"';;
+  *Labels.cmi) echo ' -pp "$AWK -f $SRCDIR/expand_module_aliases.awk"';;
   *Labels.cm[ox]) echo ' -nolabels -no-alias-deps';;
   stdlib__Float.cm[ox]) echo ' -nolabels -no-alias-deps';;
   stdlib__Oo.cmi) echo ' -no-principal';;
+  stdlib__Sys.cm[ox]) echo ' -cmi-file stdlib__Sys.cmi';;
     # preserve structure sharing in Oo.copy (PR#9767)
```

Comment:
I wonder why we need the Sys line, and it should go below the comment that is about Oo.

Reply:
It is explained [below](https://github.com/ocaml/ocaml/pull/12203#issuecomment-1522832365). Fixed the ordering of the lines. Thanks.